content/docs/quickstart/_index.md

```diff
@@ -144,6 +144,20 @@ Please make sure you have `kubectl` installed (see [kubernetes.io/docs/tasks/too
 
 :rocket: Keptn is now ready to be used.
 
+### 3. Access Keptn Bridge
+
+By default, basic authentication is in place that protects the Keptn Bridge. 
+
+* To get the `user` and `password`, execute the command:
+
+    ```console
+    keptn configure bridge --output
+    ```
+
+* Depending on the chosen way of exposing Keptn (please see [here](../0.7.x/operate/install/#install-keptn)), you need to derive the endpoint Keptn is running on. If you follow this quickstart, the endpoint is: `http://localhost:8080`
```

Comment:
We should add that it is posible to use `keptn status` to figure out the current API endpoint address:
```console
$ keptn status
Starting to authenticate
Successfully authenticated against the Keptn cluster https://keptn.example.com/api
Using a file-based storage for the key because the password-store seems to be not set up.
CLI is authenticated against the Keptn cluster https://keptn.example.com/api
```
Consequently, one would need to access https://keptn.example.com

Reply:
Good idea. I added the `keptn status` as a mean for getting the endpoint.